src/diffusers/pipelines/ledits_pp/pipeline_leditspp_if.py

```diff
@@ -0,0 +1,1396 @@
+import html
```

Comment:
Generally I'm wondering a bit how much sense it makes to have a `LEditsPPPipelineIF` class -> IF is not used that much, do we think that LeditsPPIF will be used a lot?

Reply:
I don't have any strong feelings about including IF or not. We had the implementation anyway since we wanted to demonstrate that the method worked for pixel-based diffusion models as well. 
But if you feel like it won't be used, I'm happy to drop it.

Reply:
done